### ISSUE TITLE
Read more link for screen readers

### ISSUE THREAD

**Issue by shadeed** [smashingmagazine/redesign#119]
**- Do you want to request a *feature* or report a *bug*?**
An enhancement.

**- What is the current behavior?**
Focusing on "Read more" link with VoiceOver will announce it like "Read more right arrow". 

![screen shot 2017-03-18 at 9 07 23 am](https://cloud.githubusercontent.com/assets/8038659/24069846/8ac0f0a6-0bba-11e7-9aad-80f24463ca58.jpg)

**- If the current behavior is a bug, please provide the steps to reproduce.**
1. Go to home page.
2. Turn on VoiceOver.
3. Move between the articles until you reach a one with "Read more" link.

**- What is the expected behavior?**
The arrow should be hidden for SR users.

**- Please mention your operating system version and the version of your browser**
VoiceOver, Chrome, OSX.

**Comment by vitalyfriedman** [comment]
A fine little details, thanks @shadeed — added to the to-do list.

**Comment by vitalyfriedman** [comment]
Has been fixed! @shadeed
